conans/client/graph/grapher.py

```diff
@@ -177,13 +93,15 @@ def graph(self):
                 edges.append("{ from: %d, to: %d }" % (src, dst))
         edges = ",\n".join(edges)
 
-        return self._template.replace("%NODES%", nodes).replace("%EDGES%", edges)
+        result = self._template.replace("%NODES%", nodes).replace("%EDGES%", edges)
+        visjs, viscss = self._visjs_paths()
+        return result.replace("%VISJS%", visjs).replace("%VISCSS%", viscss)
```

Comment:
It would be cleaner to use a formatting style like `.format(visjs=visjs, viscss=viscss)` instead fo this replaces

Reply:
The contents of the template contains LOTS of braces {}. For this to work, all those braces have to be escaped, which is a pain. Same goes for % syntax, there are fewer, but still there are some. We need to be able to work on plain html.